src/ECER.Tests/Integration/Resources/Accounts/Communications/CommunicationRepositoryTests.cs

```diff
@@ -28,4 +31,25 @@ public async Task QueryCommunications_ById_Found()
     // Assert
     communications.ShouldHaveSingleItem();
   }
+  
+  [Fact]
+  public async Task SeenCommunications_MarkAsSeen()
```

Comment:
This probably should be MarkUnseenCommunication_AsSeen.
This test probably won't work next time as you mark the test communication as seen unless you create a new test communication every time you run this test.

Reply:
hmm the code below seems to check that the communication has not been acknowledged first. So it seems like the communication is getting "refreshed" for each test.

Reply:
Fixture.communicationId comes from RegistryPortalWebAppScenarioBase , GetOrAddCommunication and seems we don't check that there

Reply:
@farzadnadiri @SoLetsDev this test is passing consistently